core/src/trezor/ui/layouts/tt/__init__.py

```diff
@@ -694,6 +694,7 @@ async def confirm_text(
     br_code: ButtonRequestType = ButtonRequestType.Other,
     icon: str = ui.ICON_SEND,  # TODO cleanup @ redesign
     icon_color: int = ui.GREEN,  # TODO cleanup @ redesign
+    hold: bool = False,  # TODO Only supported if NOT paginating
```

Comment:
Let me know if you want me to add support for `hold=True` when paginating too (it's just one line of code, but it won't be tested).

Reply:
let's not add functionality that we don't need. the current state of code is OK